src/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     add_compile_options(/w)
 endif()
 
+# Enable/disable optional modules compilation
```

Comment:
Since this option is related to the ModuleManager, could we move it to its corresponding cmake?

Reply:
Done